crates/language/src/php.rs

```diff
@@ -63,17 +62,21 @@ impl Language for Php {
     }
 
     fn language_name(&self) -> &'static str {
-        "Php"
+        "PhpWithHTML"
     }
     fn snippet_context_strings(&self) -> &[(&'static str, &'static str)] {
         &[
             ("", ""),
             ("", ";"),
-            ("$", ";"),
-            ("class GRIT_CLASS {", "}"),
-            ("class GRIT_CLASS { ", " function GRIT_FN(); }"),
-            (" GRIT_FN(", ") { }"),
-            ("[", "];"),
+            ("<?php ", " ?>"),
```

Comment:
These should be computable by wrapping every snippet_context_strings from phpOnly with `<?php`